src/app/services/logger/exui-logger/exui-logger.component.ts

```diff
@@ -0,0 +1,66 @@
+import {Component, OnInit, OnDestroy} from '@angular/core';
```

Comment:
this needs to be moved to app/component/exu-logger

Reply:
Done